### DISCUSSION TITLE
What CRS is Overture using?

### DISCUSSION THREAD

**Discussion by Spothedog1** [OvertureMaps/data#68] [Discussion]
Hello - To preface I'm a noob at geospatial data. My question is 

What CRS are Overture geometries in? I know it's in a geographic CRS. My best guess is WGS 84.

The reason I ask is because I loaded a subset of buildings into a GeoPandas dataframe and ran `gdf.crs` and it returned None while other datasets I've loaded into GeoPandas have return comprehensive CRS info.

**Comment by mtravis** [comment]
The CRS is WGS 84. The data itself isn't served as Geoparquet but parquet so the geometry is stored as WKB but it's possible geopandas won't read the CRS. 

If you open convert to Geoparquet and open in QGIS it does project as EPSG:4326

**Reply by Spothedog1** [reply]
Thanks for the answer. I converted it from parquet to a Pandas Dataframe via DuckDB and then to a GeoPandas Dataframe.  It probably got lost somewhere along the way.

```
Parquet -> WKB
DuckDB -> Blob
Pandas -> bytearray ->  bytes -> Shapely.WKB
GeoPandas -> Geometry
```

**Reply by mtravis** [reply]
No problem. The buildings and places data is available here as Geoparquet https://beta.source.coop/cholmes/overture
